packages/xo-server-sdn-controller/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "xo-server-sdn-controller",
+  "description": "Create the pool-wide and cross-pool private networks",
```

Comment:
```suggestion
  "description": "Creates pool-wide and cross-pool private networks",
```

Reply:
You can ask @benjamreis for help to find the best description for this one 🙂

Reply:
@pdonias seems fine to me.